internal/transport/http2_client.go

```diff
@@ -862,6 +869,12 @@ func (t *http2Client) NewStream(ctx context.Context, callHdr *CallHdr) (*Stream,
 			sh.HandleRPC(s.ctx, outHeader)
 		}
 	}
+	if transportDrainRequired {
+		if logger.V(logLevel) {
+			logger.Infof("t.nextID > MaxStreamID. transport: draining")
```

Comment:
"transport" is supposed to be the prefix of the whole log message.  How about `transport: t.nextID > MaxStreamID.  Draining`?